packages/mobile/src/recipients/RecipientItem.tsx

```diff
@@ -14,7 +14,11 @@ interface Props {
 
 class RecipientItem extends React.PureComponent<Props> {
   onPress = () => {
-    this.props.onSelectRecipient(this.props.recipient)
+    const recipient = this.props.recipient
```

Comment:
With the change on the other file it suffices for the purpose of this bug, but I thought having this here might avoid similar problems in the future.

Reply:
This would be completely redundant with the other `.toLowercase` operation right?

Instead of having two checks like this, can we instead have one happen on the import/filtering of payment requests? I think that gives us the added benefit of helping users "recover" old payment requests that were getting incorrectly filtered out before

Reply:
For the purpose of payment requests it's redundant, yes, but this is used for sending as well and might be used for other things, I just thought that making sure that addresses are lowercase is not a bad thing and could possibly avoid issues in the future.

I thought of recovering old payment requests, but if they didn't show up before I think it might be a bit weird for users for requests to suddenly pop up out of nowhere when it didn't before, especially if they sent several ones when it didn't work. The number of users is quite low for now anyways, so I guess it doesn't matter much either way.

At the same time, I'm not sure how possible it is to do it on the other side. Right now it's doing a simple `.equalTo(address)`. Firebase doesn't have a lot of flexibility in the operations it allows and this would require for them to accept a case insensitive equals, which doesn't seem supported from a quick look.

Reply:
Fair points! In that case, should we lowercase all addresses instead of only when `recipient.kind === RecipientKind.Address`?

Reply:
Yup, for some reason thought that the address field is only populated when the kind is address, but better to map it always :)